### PR TITLE
Home design unfinish

### PR DESCRIPTION
**Pull request by Alfred024**
Hola, aún no he terminado, ni tiene responsive, y cree varios  archivos que no están bien estructurados, pero se ve el diseño de lo que sería el Home. Mañana le sigo.

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -11,11 +11,17 @@
...
   --secondary-blue: #88ccff;
   --thirt-blue: #0077cc;
+  --main-white: #FFFFFF;
+  --light-gray: #f0efef;
 }
 
 * {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
```
Comment (luislealdev) [inline src/app/globals.css]: Delete this box-sizing please, if you use it you can create a class.

File: src/app/globals.css [hunk 2]
```diff
@@ -24,6 +30,31 @@ textarea {
   resize: none;
 }
 
+.absolute{
+  position: absolute;
+}
+.justify-between{
```
Comment (luislealdev) [inline src/app/globals.css]: This class already exists in leal styles, you don't need it here.

File: src/app/globals.css [hunk 3]
```diff
@@ -24,6 +30,31 @@ textarea {
...
+  justify-content: space-between;
+}
+
+.width-100{
```
Comment (luislealdev) [inline src/app/globals.css]: This class already exists in leal-styles, delete it

File: src/app/globals.css [hunk 4]
```diff
@@ -24,6 +30,31 @@ textarea {
...
+}
+
+.bg-secondary-blue {
+  background-color: var(--secondary-blue);
+}
+
+/* .bg-light-gray{
+  background-color: var(--light-gray);
+} */
+.padding-container{
```
Comment (luislealdev) [inline src/app/globals.css]: This padding container is a class that can be replaced for p-10 (For padding with 10px) in leal-styles.

File: src/app/globals.css [hunk 5]
```diff
@@ -33,10 +64,31 @@ textarea {
...
+.bg-thirt-blue {
+  background-color: var(--thirt-blue);
+}
+
 .bg-light-gray-color {
   background-color: var(--light-gray-color);
 }
 
+/* BORDER RADIUS */
+.border-radius-10{
```
Comment (luislealdev) [inline src/app/globals.css]: This is a class that already exists in leal-styles with "radius"

File: src/app/globals.css [hunk 6]
```diff
@@ -33,10 +64,31 @@ textarea {
...
+  border-radius: 10px;
+}
+.border-radius-15{
+  border-radius: 15px;
+}
+.border-radius-20{
+  border-radius: 20px;
+}
+.border-radius-30{
```
Comment (luislealdev) [inline src/app/globals.css]: This is a class that already exists in leal-styles with "radius-30", also name says 30 but value is 40.

File: src/app/globals.css [hunk 7]
```diff
@@ -193,158 +244,23 @@ textarea {
...
-.login-box a span:nth-child(4) {
-  bottom: -100%;
-  left: 0;
-  width: 2px;
-  height: 100%;
-  background: linear-gradient(360deg, transparent, #5599dd);
-  animation: btn-anim4 1s linear infinite;
-  animation-delay: .75s
+/* SPan Section */
+.Span-Section{
```
Comment (luislealdev) [inline src/app/globals.css]: This class only contains one atribute (space-between), it an be replaced with class space-between in leal-styles

File: src/app/page.tsx [hunk 8]
```diff
@@ -1,108 +1,114 @@
...
+              <p className="f-size-20">10K</p>
             </div>
-            <h3 className='f-size-30 mt-20'>Sitios web</h3>
-            <p className='f-size-24'>Desarrollamos sitios web a la medida de tus necesidades.</p>
           </div>
-          <div>
-            <div className='center-text f-size-100'>
-              <i className="fa-solid fa-cart-shopping"></i>
+
+          <div className="border-radius-30 p-10 bg-light-gray-color flex  align-center" style={{width: "180px", height:"75px"}}>
```
Comment (luislealdev) [inline src/app/page.tsx]: Try not to use fiex widths and heights, it is necessary in some cases but it is not good for automatic responsive.

File: src/components/project-card/ProjectCard.css [hunk 9]
```diff
@@ -0,0 +1,79 @@
+/* .Project-Card{
```
Comment (luislealdev) [inline src/components/project-card/ProjectCard.css:1]: Be carefull with static heights

**Review by luislealdev — CHANGES_REQUESTED** [review]
Hi, in the code i leave you some comments, if you can fixed this things it would be good. <3

**Comment by Alfred024** [comment]
I fixed the issues in the CSS.

**Pull request merged**